stdlib/poplib.pyi

```diff
@@ -51,14 +52,19 @@ class POP3:
     def stls(self, context: ssl.SSLContext | None = None) -> bytes: ...
 
 class POP3_SSL(POP3):
-    def __init__(
-        self,
-        host: str,
-        port: int = 995,
-        keyfile: str | None = None,
-        certfile: str | None = None,
-        timeout: float = ...,
-        context: ssl.SSLContext | None = None,
-    ) -> None: ...
+    if sys.version_info >= (3, 12):
+        def __init__(
+            self, host: str, port: int = 995, *, timeout: float = ..., context: ssl.SSLContext | None = None
+        ) -> None: ...
+    else:
+        def __init__(
+            self,
+            host: str,
+            port: int = 995,
+            keyfile: str | None = None,
+            certfile: str | None = None,
+            timeout: float = ...,
+            context: ssl.SSLContext | None = None,
+        ) -> None: ...
     # "context" is actually the last argument, but that breaks LSP and it doesn't really matter because all the arguments are ignored
     def stls(self, context: Any = None, keyfile: Any = None, certfile: Any = None) -> NoReturn: ...
```

Comment:
The arguments were removed here as well.

Reply:
Ah, good spot. Stubtest didn't complain because the method's already allowlisted:

https://github.com/python/typeshed/blob/53144ca57202a7bc5b28fd8e48e6bf62ad1b8a5b/tests/stubtest_allowlists/py3_common.txt#L111